user/tests/wiring/i2c_master_slave/common/common.inc

```diff
@@ -0,0 +1,53 @@
+#include "random.h"
+
+#ifndef TEST_I2C_BUFFER_SIZE
+#define TEST_I2C_BUFFER_SIZE (I2C_BUFFER_LENGTH)
+#else
+
+HAL_I2C_Config USE_ACQUIRE_WIRE_BUFFER() {
+#if TEST_I2C_BUFFER_SIZE != 0
+    HAL_I2C_Config config = {
+		.size = sizeof(HAL_I2C_Config),
+		.version = HAL_I2C_CONFIG_VERSION_1,
+		.rx_buffer = new (std::nothrow) uint8_t[TEST_I2C_BUFFER_SIZE],
```

Comment:
This might be `nullptr`.

Reply:
It should be ok, because HAL will handle this case.